library/repo.go

```diff
@@ -348,6 +351,19 @@ func (r *Repo) GetPipelineType() string {
 	return *r.PipelineType
 }
 
+// GetPreviousName returns the PreviousName field.
+//
+// When the provided Repo type is nil, or the field within
+// the type is nil, it returns the zero value for the field.
+func (r *Repo) GetPreviousName() string {
+	// return zero value if Repo type or NameHsitory field is nil
```

Comment:
Should this be `PreviousName`?

```suggestion
	// return zero value if Repo type or PreviousName field is nil
```

Reply:
When you do a find and replace routine but misspell `NameHistory`, I suppose things like this happen 🙃